test/unit_test/static/quantity_test.cpp

```diff
@@ -545,6 +545,12 @@ static_assert(quantity{4} % quantity{2} == 0);
 static_assert(4 % quantity{2} == 0);
 static_assert(quantity{4} % 2 == 0);
 
+static_assert(!is_same_v<decltype((quantity{std::uint8_t(0)} + quantity{std::uint8_t(0)}).count()), int>);
+static_assert(!is_same_v<decltype((quantity{std::uint8_t(0)} - quantity{std::uint8_t(0)}).count()), int>);
+static_assert((quantity{std::uint8_t(128)} + quantity{std::uint8_t(128)}).count() !=
+              std::uint8_t(128) + std::uint8_t(128));
+static_assert((quantity{std::uint8_t(0)} - quantity{std::uint8_t(1)}).count() != std::uint8_t(0) - std::uint8_t(1));
+
```

Comment:
Why do we have inequality here rather than equality?

Reply:
I see this comment is marked as outdated. I attempted to take a TDD approach, but did it wrong. I should've used equality, watched the tests fail, and then fixed them. Rather, I used inequality and the test that shouldn't have passed did pass. Then I fixed them (and changed inequality to equality).